devicetypes/smartthings/zigbee-accessory-dimmer.src/zigbee-accessory-dimmer.groovy

```diff
@@ -42,64 +43,140 @@ metadata {
 	}
 }
 
-def getSTEP() {10}
+def getSTEP() { 10 }
+
+def getSCENES_CLUSTER() { 0x0005 }
+def getSCENES_STORE_SCENE_COMMAND() { 0x04 }
+def getSCENES_RECALL_SCENE_COMMAND() { 0x05 }
+
+def getONOFF_ON_COMMAND() { 0x01 }
+def getONOFF_OFF_COMMAND() { 0x00 }
+def getLEVEL_MOVE_LEVEL_COMMAND() { 0x00 }
+def getLEVEL_MOVE_COMMAND() { 0x01 }
+def getLEVEL_STEP_COMMAND() { 0x02 }
+def getLEVEL_STOP_COMMAND() { 0x03 }
+def getLEVEL_MOVE_LEVEL_ONOFF_COMMAND() { 0x04 }
+def getLEVEL_MOVE_ONOFF_COMMAND() { 0x05 }
+def getLEVEL_STEP_ONOFF_COMMAND() { 0x06 }
+def getLEVEL_STOP_ONOFF_COMMAND() { 0x07 }
+def getLEVEL_DIRECTION_UP() { "00" }
+def getLEVEL_DIRECTION_DOWN() { "01" }
+
+def getUINT8_STR() { "20" }
```

Comment:
Rather than defining this I think it'd be better to just convert `DataType.UINT8` to a hex string at the place you need to use it

Reply:
Hmmmmm I think I'd do that if I were getting dynamic values, but since it is only this one value (which the spec isn't going to change) I feel like it would be computationally simpler to leave it as a static declaration. It just seems like adding `String.format("%02X", value)` throws more complexity in to it. :/

Reply:
🤷‍♂ I don't care strongly. It's used in one place so a single `String.format` doesn't seem like a big deal, but I'm fine either way

Reply:
Cool. I'll leave it static, for now. I'd like to think that I'm doing my part shaving off some CPU processes which would cut down CPU load a little. 😮